azure-discovery/src/main/java/io/crate/azure/management/AzureComputeServiceImpl.java

```diff
@@ -111,14 +113,13 @@ private Configuration createConfiguration() {
                 appId,
                 appSecret
             );
-            conf = ManagementConfiguration.configure(
-                null,
-                URI.create(Azure.ENDPOINT),
-                subscriptionId, // subscription id
-                authRes.getAccessToken()
-            );
+
+            DefaultBuilder registry = DefaultBuilder.create();
+            AzureModule.registerServices(registry);
+            conf = ManagementConfiguration.configure(null, new Configuration(registry),
+                URI.create(Azure.ENDPOINT), subscriptionId, authRes.getAccessToken());
         } catch (Exception e) {
-            logger.error("Could not create configuration for Azure clients: {}", e.getMessage());
+            logger.error("Could not create configuration for Azure clients: {}", e);
```

Comment:
remove place holder as it is not used (exception is passed in here for complete stacktrace).